onnx/defs/schema.cc

```diff
@@ -584,4 +584,54 @@ size_t ReplaceAll(std::string& s, const char* from, const char* to) {
   }
   return numReplaced;
 }
+
+TensorProto::DataType datatypeFromString(const std::string& type) {
+  if (type == "FLOAT") {
+    return TensorProto::FLOAT;
+  }
+  if (type == "FLOAT") {
```

Comment:
2 FLOAT? C++ may miss some float, so we need a second check to catch them?

Reply:
float is a very important datatype and needs extra attention